internal-packages/eslint-config-next-on-pages/index.js

```diff
@@ -28,6 +28,8 @@ module.exports = {
 		"@typescript-eslint/await-thenable": "error",
 		"@typescript-eslint/no-floating-promises": "error",
 		"@typescript-eslint/promise-function-async": "error",
+
+		"deprecation/deprecation": "warn",
```

Comment:
usually I am for having all rules set to `error` (since `warn`s are a bit pointless and can easily be overlooked/ignored) but since sometimes you might be forced to call deprecated functions so I set this to `warn` (so that we can use this to check our code but it can't make linting fail) 🤔 

I'm totally happy to set it to `error` if preferred 🙂

Reply:
My take is that we should always "try" to use `error` and then opt out on a case by case basis with eslint comments in the code.

Reply:
ok good point, let me just set this to `error` here then 🙂 (also we don't have currently any deprecated call 🙂👍)